k0765065/assignment07.py

```diff
@@ -0,0 +1,10 @@
+from sys import argv
+
+if len(argv) == 3:
+    from prancercise import prancercise
```

Comment:
You should try whenever possible to place imports at the top of the file, that way it is easy to glance at the file and see all its dependencies.